.github/workflows/release-on-comment.yml

```diff
@@ -0,0 +1,26 @@
+# part of our release process, see `release-on-comment.py`
+name: release on comment
+
+on:
+  issues:
+    types: [opened, edited]
+  issue_comment:
+    types: [created, edited]
```

Comment:
Does this mean the job is run for any issue/comment being created?
Can this be limited to e.g. only for when pytestbot is mentioned?

Reply:
Yep, managed to find a way so we skip right at the beginning, before even starting up a VM. 👍